templates/bundle/Dockerfile

```diff
@@ -1,4 +1,11 @@
-FROM --platform=${TARGETPLATFORM} liferay/jdk11:latest
+FROM --platform=${TARGETPLATFORM} liferay/jdk11:latest AS osbase
```

Comment:
What is the purpose of moving the apt-get commands into a different stage?

Reply:
Stages are cache able by docker. if there is no input changes that stage is taken from cache saving huge amount of time. In this case it was 3-4 minutes.